bpf/lib/l4.h

```diff
@@ -37,7 +37,8 @@ static __always_inline int l4_modify_port(struct __ctx_buff *ctx, int l4_off,
 					  int off, struct csum_offset *csum_off,
 					  __be16 port, __be16 old_port)
 {
-	if (csum_l4_replace(ctx, l4_off, csum_off, old_port, port, sizeof(port)) < 0)
+	if (csum_off->offset > 0 &&
+	    csum_l4_replace(ctx, l4_off, csum_off, old_port, port, sizeof(port)) < 0)
```

Comment:
If the caller is already dropping SCTP before it hits this line (I'm looking at `reverse_map_l4_port()`), is this change still necessary?

Reply:
No, SCTP packets are dropped before entering all instances of this function. Probably a remnant of before we added explicit drop for SCTP packets that need to change ports.